watertap/flowsheets/electrodialysis/electrodialysis_1stack.py

```diff
@@ -266,7 +269,7 @@ def optimize_system(m, solver=None, checkpoint=None, fail_flag=True):
     # Choose and unfix variables to be optimized
     m.fs.EDstack.voltage_applied[0].unfix()
     m.fs.EDstack.cell_pair_num.unfix()
-    m.fs.EDstack.cell_pair_num.set_value(10)
+    m.fs.EDstack.cell_pair_num.set_value(30)
```

Comment:
I would summarize the changes made in the PR in the "Changes proposed in this PR:" section for tractability, and to clarify why this is changing

Reply:
This change is insignificant. Both work fine.

Reply:
I agree this change is insignificant. It only becomes significant if a user is referencing this flowsheet for something unaware to us, and their code breaks after merging with this PR. That's why at the minimum the "Fixes", "Summary", and "Changes" sections of the PR should have information regarding the changes as a history to reference.